test_programs/execution_success/derive/src/main.nr

```diff
@@ -21,15 +21,18 @@ comptime fn derive_do_nothing(s: StructDefinition) -> Quoted {
 
 // Test stdlib derive fns & multiple traits
 #[derive(Eq, Default)]
-struct MyOtherStruct {
-    field1: u32,
-    field2: u64,
+struct MyOtherStruct<A, B> {
```

Comment:
Please add a nested-struct and/or a `struct -> tuple/array/slice -> struct` example

Reply:
Done